src/govuk/components/accordion/accordion.test.js

```diff
@@ -118,13 +116,46 @@ describe('/components/accordion', () => {
         expect(expandedState).toEqual(expandedStateAfterRefresh)
       })
 
-      describe('"+/-" icon', () => {
-        it('should display "+/-" in the section headings', async () => {
+      describe('"↓/↑" icon', () => {
+        it('should display "↓/↑" in the section headings', async () => {
+          await page.goto(baseUrl + '/components/accordion/preview', { waitUntil: 'load' })
+
+          const numberOfExampleSections = 2
+          const numberOfIcons = await page.evaluate(() => document.body.querySelectorAll('.govuk-accordion .govuk-accordion__section .govuk-accordion-nav__chevron').length)
+
+          expect(numberOfIcons).toEqual(numberOfExampleSections)
+        })
+      })
+
+      describe('hidden comma', () => {
+        it('should contain hidden comma " ," for assistive technology', async () => {
+          await page.goto(baseUrl + '/components/accordion/preview', { waitUntil: 'load' })
+
+          const numberOfExampleSections = 2
+          const numberOfCommas = await page.evaluate(() => document.body.querySelectorAll('.govuk-accordion .govuk-accordion__section .govuk-visually-hidden:first-child').length)
+
+          expect(numberOfCommas).toEqual(numberOfExampleSections)
+        })
+      })
+
+      it('should change the (combined) Show[ this section] text to Hide[ this section ] when sections are opened', async () => {
+        await page.goto(baseUrl + '/components/accordion/preview', { waitUntil: 'load' })
+
+        const ShowOrHideButtonText = await page.evaluate(() => document.body.querySelector('.govuk-accordion__section-toggle-text').textContent)
+
+        await page.click('.govuk-accordion .govuk-accordion__section:nth-of-type(2) .govuk-accordion__section-header')
+        await page.click('.govuk-accordion .govuk-accordion__section:nth-of-type(3) .govuk-accordion__section-header')
+
+        expect(ShowOrHideButtonText).toEqual('Hide this section')
+      })
+
```

Comment:
There's a bit of a (pre-existing) gap in the tests, currently there are no tests for an accordion with additional description/summary text.

Given that this PR adds a bit more sophisticated handling of the additional description, it might be worth adding tests such as `it('should move the additional description to the button text')`; would that be okay?

Reply:
Make sense, i've added a tests to check for the span being changed. However it might be worth filling that gap to add the missing check of the summary text just existing.